emitter/dir.go

```diff
@@ -32,6 +32,10 @@ type Dir struct {
 	paths   map[string]int
 }
 
+func unknownFormat(format string) error {
```

Comment:
pre-existing so not for this PR but this should probably use the new 1.13 wrapped errors so that the underlying error is recognizable...

Reply:
ok